packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/fastmcp_tools/search.py

```
@@ -0,0 +1,181 @@
+
+    @mcp.tool(annotations={"readOnlyHint": True})
+    async def attachment_search(
+        ctx: Context,
+        query: Annotated[
+            str, Field(description="The search query in natural language", min_length=1)
+        ],
+        attachment_filter: Annotated[
+            AttachmentFilter | None,
+            Field(description="Attachment filters"),
+        ] = None,
+        include_parent_context: Annotated[
```

Ruff (0.15.20)

[warning] Boolean default positional argument in function definition

(FBT002)